CMakeLists.txt

```diff
@@ -453,6 +453,15 @@ if(APPLE)
 		src/platform/macos/utils.h)
 endif()
 
+if(CMAKE_SYSTEM_NAME STREQUAL "Emscripten")
+	option(PLAYER_JS_BUILD_SHELL "Build the Player executable as a shell file (.html) instead of a standalone javascript file (.js)" OFF)
+	set(PLAYER_JS_PREJS "${CMAKE_CURRENT_SOURCE_DIR}/resources/emscripten/emscripten-pre.js" CACHE STRING "File to use for --pre-js")
+	set(PLAYER_JS_POSTJS "${CMAKE_CURRENT_SOURCE_DIR}/resources/emscripten/emscripten-post.js" CACHE STRING "File to use for --post-js")
+	set(PLAYER_JS_SHELL "${CMAKE_CURRENT_SOURCE_DIR}/resources/emscripten/emscripten-shell.html" CACHE STRING "Shell file to use (--shell-file). Requires PLAYER_JS_BUILD_SHELL=ON")
+	set(PLAYER_JS_GAME_URL "games/" CACHE STRING "Game URL/directory where the web player searches for games")
+	set_property(SOURCE src/async_handler.cpp APPEND PROPERTY COMPILE_DEFINITIONS "EM_GAME_URL=\"${PLAYER_JS_GAME_URL}\"")
```

Comment:
This is a nice feature. For autotools this involves either a convenience lib or manual makefile rules, which is both ugly.
I would suggest we remove the `EM_GAME_URL` completely from there, since the additional steps are not implemented as well.

Reply:
did this now